packages/core/helper-plugin/src/hooks/useFocusWhenNavigate.ts

```diff
@@ -1,8 +1,15 @@
-import { useEffect } from 'react';
+import React, { DependencyList } from 'react';
+interface UseFocusWhenNavigateProps {
+  selector?: string;
+  dependencies?: DependencyList;
+}
```

Comment:
Not sure if you've update since: https://github.com/strapi/strapi/pull/17737#discussion_r1302801189

I think we are going for:

```suggestion
import * as React from 'react';
interface UseFocusWhenNavigateProps {
  selector?: string;
  dependencies?: React.DependencyList;
}
```